docs/se/guides/05_metrics.adoc

```diff
@@ -245,7 +245,7 @@ import jakarta.json.Json;
 import jakarta.json.JsonBuilderFactory;
 import jakarta.json.JsonObject;
 import org.eclipse.microprofile.metrics.Counter;  // <1>
-import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.MetricRegistry; // <1>
```

Comment:
(Duplicate callout number?)

Reply:
Intentional. The callout at the bottom of the example is `<1> Import metrics classes.` and there are two metrics classes imported.